### ISSUE TITLE
Fold interlibrary angle symbol

### ISSUE THREAD

**Issue by JEAndrew** [StraboSpot/strabo-mobile#414]
The kobo form uses the degree symbol but it shows up as a diamond symbol with a question mark in the app

**Comment by jessicagood** [comment]
Yeah there were some symbols that I couldn't get to display properly. I'm not sure exactly which field you're referring to here. I can't find the symbols displayed as diamonds that I had see before. If you are able to change the KoBo form so it doesn't use special symbols let me know so I can update the app.

This is related to https://github.com/StraboSpot/strabo-mobile/issues/115
